frontend/src/pages/IstioConfigNew/IstioConfigNewPage.tsx

```diff
@@ -86,27 +95,28 @@ import { Labels } from '../../components/Label/Labels';
 import { WizardLabels } from '../../components/IstioWizards/WizardLabels';
 
 type Props = {
```

Comment:
Since some of the component props are Redux ones, you should split them in two types:
```
type ReduxProps = {
   activeClusters: MeshCluster[];
  activeNamespaces: Namespace[];
  namespacesPerCluster?: Map<string, string[]>;
}

type Props = ReduxProps & {
  objectType: string;
}
```
Then set `Redux` props as return type of `mapStateToProps` methods.

```
const mapStateToProps = (state: KialiAppState): ReduxProps => {
  return {
    activeClusters: activeClustersSelector(state),
    activeNamespaces: activeNamespacesSelector(state),
    namespacesPerCluster: namespacesPerClusterSelector(state)
  };
};
```